README.md

```diff
@@ -196,6 +197,60 @@ await client.time()
 await client.close()
 ```
 
+### Using the Realtime API
+The python realtime API currently only supports authentication with ably API key.
+#### Creating a client
+```python
+from ably import AblyRealtime
+
+async def main():
+    client = AblyRealtime('api:key')
+    channel = client.channels.get('channel_name)
+```
+
+#### Subscribing to a channel for event
+```python
+message_future = asyncio.Future()
+
+def listener(message):
+    message_future.set_result(message)
+
+channel.subscribe('event', listener)
```

Comment:
```suggestion
# Subscribe to messages with the 'event' name
await channel.subscribe('event', listener)
```

Reply:
you should also add something to mention that `channel.subscribe` is an coroutine function and will resolve when the channel is attached.